### PR TITLE
Add a newsletter for 2019-10-31

### PR DESCRIPTION
**Pull request by notlee**
Content is ready for review and any additional suggestions! 

Todo when the content is approved:
- [x] ensure correct links to the major cascade blogpost (if it's published)
- [x] update image links to use absolute urls and the image service
- [x] make the email copy

### CODE REVIEW

File: _posts/2019-10-31-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  - Newsletter
+---
+
+<abbr title="Too long; didn't read">
+	<strong>
+	TL;DR:
+	</strong>
+</abbr> {{page.description}}
+
+## Top things
```
Comment (notlee) [inline _posts/2019-10-31-newsletter.md]:
```suggestion
## Top Things
```

File: _posts/2019-10-31-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	</figcaption>
+</figure>
+
+If your team maintains an Origami compatible component which doesn't already use our new Continuous Integration tools let us know, we're happy to help you start using them.
+
+## Special Thanks
+
+Our special thanks this month goes to everyone who has contributed to the design basics working group. Thanks Luke Griffiths, Caroline Nevitt, Mark Limb, Simon Coxon, and Jessica Engerer! They helped rationalise inconsistent design in typography and buttons for the upcoming [major releases](/2019/10/31/major-cascade/) discussed above, and have created a Sketch UI Kit of Origami components. Learn more about the new [UI Kit in Luke Griffiths' blog post](https://medium.com/ft-product-technology/ft-design-basics-1-why-we-finally-built-a-ui-kit-850e98b127bf).
+
+## Broader update
```
Comment (notlee) [inline _posts/2019-10-31-newsletter.md]:
```suggestion
## Broader Update
```

File: _posts/2019-10-31-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Top Things
+
+These are some of the bigger things we've done over the last three months since our previous newsletter...
+
+### Major Component Releases
+
+We've been working on upgrades to fundamental Origami components like o-typography, o-colors, and o-buttons. Releasing a new version of these components necessitates a major release of other components too. So we've taken the opportunity to introduce recent proposals across all our 52 components. The upgrades focus on:
+
+- Improving our ability iterate on and share editorial styles across projects.
```
Comment (JakeChampion) [inline _posts/2019-10-31-newsletter.md]:
```suggestion
- Improving our ability to iterate on and share editorial styles across projects.
```

File: _posts/2019-10-31-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+	</strong>
+</abbr> {{page.description}}
+
+## Top Things
+
+These are some of the bigger things we've done over the last three months since our previous newsletter...
+
+### Major Component Releases
+
+We've been working on upgrades to fundamental Origami components like o-typography, o-colors, and o-buttons. Releasing a new version of these components necessitates a major release of other components too. So we've taken the opportunity to introduce recent proposals across all our 52 components. The upgrades focus on:
```
Comment (rowanmanning) [inline _posts/2019-10-31-newsletter.md]:
```suggestion
We've been working on upgrades to fundamental Origami components like o-typography, o-colors, and o-buttons. Releasing a new version of these components necessitates a major release of other components too. So we've taken the opportunity to introduce recent proposals across all 52 of our components. The upgrades focus on:
```

**Comment by notlee** [comment]
Merging this now but will send the actual email tomorrow morning

**Pull request merged**